core/src/main/scala/org/apache/spark/rdd/JdbcRDD.scala

```diff
@@ -151,7 +154,10 @@ object JdbcRDD {
    *   The RDD takes care of closing the connection.
    * @param sql the text of the query.
    *   The query must contain two ? placeholders for parameters used to partition the results.
+   *
+   *   {{{
    *   E.g. "select title, author from books where ? <= id and id <= ?"
+   *   }}}
```

Comment:
Same comment on the rest, ideally, can we do this?

For example: `select title ... <= ?`

or at best

For example:
```
select title ... <= ?
```